docs/shared/getting-started/why-nx.md

```diff
@@ -0,0 +1,29 @@
+# Why Nx?
+
+We created Nx because developers struggle to configure, maintain and especially integrate various tools and frameworks. Setting up a system that works well for a handful of developers and at the same time, easily scales up to an entire organization is hard. This includes setting up low-level build tooling, configuring fast CI, and keeping your codebase healthy, up-to-date, and maintainable.
+
+We wanted to provide a solution that is easy to adopt and scales.
+
+## How Does Nx Help You?
+
+In a nutshell, Nx helps to: a) **speed up your computation** (e.g. builds, tests etc), locally and on CI and b) to **integrate and automate your tooling** via its plugins. All of this can be adopted incrementally. You can, but you don't have to use plugins. Look at the Nx architecture in the next section to learn more.
```

Comment:
If you're using (a) and (b) in a paragraph, it should be an actual list instead.

Reply:
You can, but you don't have to use plugins. => You have the choice to use plugins or not.

Reply:
> You can, but you don't have to use plugins. => You have the choice to use plugins or not.

What about: "You can use plugins, but you don't have to."

I wanted to emphasize the fact that plugins are optional. Because that's such a common misconception